reference/orders.v3.yml

```
@@ -416,6 +416,50 @@ paths:
         required: true
         schema:
           type: string
+  '/stores/{store_hash}/v3/orders/payment_actions/refunds/{refund_id}':
+    get:
+      summary: Return a refund by refund ID.
+      description: |-
+        Returns a refund by refund ID.
+        Requires at least one of the following scopes:
+        * `store_v2_orders`
+        * `store_v2_transactions`
+      operationId: RefundID_Get
+      parameters:
+        - name: refund_id
+          in: path
+          description: Refund ID.
+          required: true
+          schema:
+            type: integer
+        - name: store_hash
+          in: path
+          required: true
+          schema:
+            type: string
+        - $ref: '#/components/parameters/Accept'
+        - $ref: '#/components/parameters/Content-Type'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/RefundID_Get'
+        required: true
+        x-examples: null
+      responses:
+        '201':
+          description: Return a refund with 'Refund_ID'.
```

```suggestion
          description: Return a refund by refund ID.
```

I think this line can be removed